.cargo/config.toml

```diff
@@ -25,6 +25,10 @@ rustflags = ["-C", "target-feature=+crt-static"]
 [target.aarch64-pc-windows-msvc]
 rustflags = ["-C", "target-feature=+crt-static"]
 
+# Enable jsconv target feature for all aarch64 targets to support FJCVTZS instruction
+[target.'cfg(target_arch = "aarch64")']
+rustflags = ["-C", "target-feature=+jsconv"]
```

Comment:
Wouldn't this enable the compiler to use `FJCVTZS` anywhere and cause crash on Arm v8-A machines? Did you meant to set this for `target.aarch64-apple-darwin`?

Reply:
I see. I didn't know that. It seems there's `#[target_feature(enable = "jsconv")]` and that seems to be the recommended way.
https://doc.rust-lang.org/beta/std/arch/index.html#dynamic-cpu-feature-detection:~:text=Using%20a%20compiler,the%20one%20function.